api/udpa/data/orca/v1/orca_load_report.proto

```diff
@@ -22,10 +22,13 @@ message OrcaLoadReport {
   // during the request.
   double mem_utilization = 2 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
 
+  // Total QPS being served by an endpoint.
+  uint64 qps = 3;
```

Comment:
nit: I would probably go with "rps" which I think is more of an industry accepted term for this.